apps/api/src/config/cors.spec.ts

```diff
@@ -0,0 +1,90 @@
+import { corsOptionsDelegate } from './cors';
+import { spy } from 'sinon';
+import { expect } from 'chai';
+
+describe('CORS Configuration', () => {
+  describe('Local Environment', () => {
+    beforeEach(() => {
+      process.env.NODE_ENV = 'local';
+    });
+
+    afterEach(() => {
+      process.env.NODE_ENV = 'test';
+    });
+
+    it('should allow all origins', () => {
+      const callbackSpy = spy();
+
+      // @ts-ignore
+      corsOptionsDelegate({ url: '/v1/test' }, callbackSpy);
+
+      expect(callbackSpy.calledOnce).to.be.ok;
+      expect(callbackSpy.firstCall.firstArg).to.be.null;
+      expect(callbackSpy.firstCall.lastArg.origin).to.equal('*');
+    });
+  });
+
+  for (const environment of ['dev', 'production']) {
```

Comment:
```suggestion
  (['dev', 'production'] as const).forEach((environment) => {
```
<!-- Suggestions are specific requests to improve the subject under review. It is assumed that we all want to do what's best, so these comments are never dismissed as “mere suggestions”, but are taken seriously.  -->
**🎯 suggestion:** ‏I like this way because it emphasizes the values being tested and the `as const` removes the necessity for type-casting later!